sdk/search/azure-search-documents/src/main/java/com/azure/search/documents/SearchAsyncClient.java

```diff
@@ -189,14 +188,13 @@ HttpPipeline getHttpPipeline() {
      */
     @ServiceMethod(returns = ReturnType.SINGLE)
     public Mono<Response<IndexDocumentsResult>> uploadDocumentsWithResponse(Iterable<?> documents,
-        IndexDocumentsOptions options, RequestOptions requestOptions) {
-        return withContext(context -> uploadDocumentsWithResponse(documents, options, requestOptions, context));
+        RequestOptions requestOptions) {
```

Comment:
You removed `IndexDocumentOptions`

Reply:
This goes for all document indexing APIs.

Not sure how tests aren't failing due to this.

Reply:
Will double check. That's a miss after merge conflicts.

Reply:
Do you have tests on throwOnError to false?

Reply:
There are a few test that are setting the value, don't remember if it was true or false, but it should trigger a compile error.

Reply:
Will add tests for the senarios.